reset.sh

```diff
@@ -1,4 +1,7 @@
 #!/bin/bash
 
-dropdb --if-exists boilerplateapp
-diesel setup --database-url ${BOILERPLATEAPP_DATABASE_URL}
+# Source .env to set local variables
+eval $(echo "$(cat .env) $1" | tr '\n' ' ')
```

Comment:
We won't need this weirdness if you write `export` in front of the variables in the .env file above. Then you can make this just `source .env`.